go-controller/pkg/ovn/endpoints.go

```diff
@@ -210,21 +211,21 @@ func (ovn *Controller) handleExternalIPs(svc *kapi.Service, svcPort kapi.Service
 	if len(svc.Spec.ExternalIPs) == 0 {
 		return
 	}
-	for _, extIP := range svc.Spec.ExternalIPs {
-		lb := ovn.getDefaultGatewayLoadBalancer(svcPort.Protocol)
-		if lb == "" {
-			klog.Warningf("No default gateway found for protocol %s\n\tNote: 'nodeport' flag needs to be enabled for default gateway", svcPort.Protocol)
-			continue
-		}
-		if removeLoadBalancerVIP {
+	lb := ovn.getDefaultGatewayLoadBalancer(svcPort.Protocol)
+	if lb == "" {
+		klog.Warningf("No default gateway found for protocol %s\n\tNote: 'nodeport' flag needs to be enabled for default gateway", svcPort.Protocol)
+	}
```

Comment:
@danwinship isn't this going to fall through and try to either create/delete the LB VIPs with an empty LB UUID?

Reply:
yup. fixed